<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>929</width>
    <height>429</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QTableWidget" name="tableWidget">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>70</y>
     <width>931</width>
     <height>361</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QTableWidget {
    background-color: #1f2937;      /* body gelap */
    color: #f1f5f9;                 /* text isi tabel */
    gridline-color: #374151;
    selection-background-color: #2563eb;
    selection-color: white;
}

/* Header kolom */
QHeaderView::section {
    background-color: #111827;      /* header lebih gelap */
    color: #f9fafb;                 /* text header putih */
    padding: 6px;
    border: none;
    font-weight: bold;
}

/* Alternating row */
QTableWidget::item:alternate {
    background-color: #111827;
}

/* Hover row */
QTableWidget::item:hover {
    background-color: #374151;
}</string>
   </property>
   <column>
    <property name="text">
     <string>Nama Pengguna</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Aktivitas</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Keterangan</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Tabel Terkait</string>
    </property>
   </column>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>361</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>16</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(230, 230, 230);</string>
   </property>
   <property name="text">
    <string>Log Aktivitas Pengguna</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_report_aktivitas_user">
   <property name="geometry">
    <rect>
     <x>770</x>
     <y>10</y>
     <width>151</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
	background-color: rgb(85, 255, 127);
    border: none;
    text-align: center;
    font-size: 10pt;
    border-radius: 6px;  /* Radius sedang, tidak terlalu bulat */
    margin: 2px 8px;
    color: rgb(0,0,0);
}
QPushButton:hover {
	background-color: rgb(170, 255, 127);
}

</string>
   </property>
   <property name="text">
    <string>Cetak</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
